app/views/users/index.html.erb

```diff
@@ -15,7 +15,7 @@
           <div class="col-xs-12 col-sm-4">
           <%= select_tag :project_filter, options_from_collection_for_select(@projects, 'id', 'title', params[:project_filter]), {prompt: 'Project involvement...', class: 'form-control'} %>
           </div>
-          <% if @current_user && @current_user.latitude %>
+          <% if current_user && current_user.latitude %>
```

Comment:
The same here, if we want to get the `current_user` this is the helper we should be using.